config/routes.rb

```diff
@@ -84,7 +84,7 @@
 
   namespace :lead_providers, path: "lead-providers" do
     resources :your_schools, path: "/your-schools", only: %i[index create]
-    resources :school_details, path: "school-details", only: %i[show]
+    resources :partnerships, only: %i[show]
```

Comment:
Do we need to change any cypress tests for that? I kind of would expect that there would be a test saying 'given something, when I go to partnership page, then I am on a page with path matching xxx'.

Reply:
Soooo, that's quite interesting discussion - should feature tests at all care about urls? Do users care?

The current test for `your-schools` does not check for the urls. Since this refactor does not change the user experience, I'd vote that we should not check for the urls in the tests. I think the only reason we got sort of used to check the urls is the concurrent nature of capybara when we used to check for url to ensure that page has loaded after form submission...

Reply:
I think feature tests care about urls because it's a relatively easy and resilient way of checking that 'when I click a button I go to the page that that button should lead to'. If we don't have a test like that for the page where you can see a partnership, well, no need to worry about it.